docs/_docs/04_community/contributing.md

```diff
@@ -150,16 +149,57 @@ to include the following:
    test output so we can verify that everything is working.
 1. Any notes on backwards incompatibility or downtime.
 
+#### Validate the Pull Request for Azure Platform
+
+If you're contributing code for the [Azure Platform](https://azure.com), please follow the below guidelines after [creating a pull request](https://help.github.com/articles/creating-a-pull-request/):
+
+##### Run the Azure CI Pipeline on your forked repo
+
+We have a separate CI pipeline for Azure code. To run it on a forked repo:
+
+1. Run the following [Azure Cli](https://docs.microsoft.com/cli/azure/) command on your preferred Terminal to create Azure credentials and copy the output:
+
+
+    ```bash
+    az ad sp create-for-rbac --name "terratest-az-cli" --role contributor --sdk-auth
+    ```
+
+1. Go to Secrets page ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets](https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets)) under Settings tab ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings](https://github.com/<GITHUB_ACCOUNT>/terratest/settings)) on GitHub
+
+1. Create a new `Secret` named `AZURE_CREDENTIALS` and paste the Azure credentials you copied from the 1<sup>st</sup> step as the value
+
+1. Create a [new Personal Access Token (PAT)](https://github.com/settings/tokens/new) page under [Settings](https://github.com/settings/profile) / [Developer Settings](https://github.com/settings/apps), making sure `write:discussion` scope is checked. Click the _Generate token_ button and copy the generated PAT.
+

```

Comment:
remove extra linebreaks (md linter in vscode should show these.

Reply:
removed, thanks...